exe/etcd-env

```diff
@@ -40,10 +43,17 @@ def fetch(uri_str, limit = 10)
 end
 
 if __FILE__ == $PROGRAM_NAME
```

Comment:
This is to check if it's being run as a script, not being required by some
other script. Not quite sure how to do that for a gem...